### ISSUE TITLE
Dynamic content not rendered in Shoes Manual

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#325]
SCENARIO:
* Open Shoes Manual
* Search for **colors**
* Click on _Colors_ section

(Same goes for **sample** as a keyword)

![image](https://cloud.githubusercontent.com/assets/8844578/23179920/a1750c6e-f83d-11e6-8ac3-8797aedcd652.png)

The proper rendering can be seen in **Shoes Manual > Shoes > Colors**.

**Comment by ccoupe** [comment]
Depends on the section/subsection of the search item chosen. Methods work fine.
![active-content](https://cloud.githubusercontent.com/assets/222691/23293933/36e2392c-fa26-11e6-9be5-39c2b565c616.png)

**Comment by BackOrder** [comment]
You are correct. This is handled in [`help.rb#wiki_tokens`](https://github.com/Shoes3/shoes3/blob/master/lib/shoes/help.rb#L78) and [`help.rb#dewikify`](https://github.com/Shoes3/shoes3/blob/master/lib/shoes/help.rb#L111). For some reason it won't dewikify on search results.

Also, it's kinda interesting because we had a long conversation about being able to run code in Shoes eBook for exactly that kind of purpose. Shoes Manual already do it.
